### PR TITLE
Tuesday October 6 2020 Meeting

### PR DESCRIPTION
**Pull request by nkohen**
Please comment on this PR or DM with things we should discuss.

Once the meeting has happened I will add meeting notes and then once they have been reviewed this will be merged into meetings directory.

### CODE REVIEW

File: meeting/meeting_october_6_2020.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  * https://github.com/discreetlogcontracts/dlcspecs/pull/81
+* Static test vector PR nearing completion
+  * https://github.com/discreetlogcontracts/dlcspecs/pull/100
+* Initial proposal for on-chain/non-interactive handling protocol
+  * ariard has received and responded to initial review
+  * https://github.com/discreetlogcontracts/dlcspecs/pull/87
+* Lloyd will write a doc like BIP 340 describing our variant of ECDSA Adaptor Signatures
+* Oracle Specification
+  * What should be merged?
+* Using SIGHASH_SINGLE (or other) for winner-take-all cases?
```
Comment (benthecarman) [inline meeting/meeting_october_6_2020.md]: In the issue me and @ariard came to the conclusion we could use `SIGHASH_SINGLE` for all cases actually, and then maybe `SIGHASH_NONE | ANYONECANPAY` for the winner-take-all cases.

**Comment by ghost** [comment]
I will be trying [NDLC-CLI](https://github.com/dgarage/NDLC) and will have few things to discuss about [p2p derivatives](https://github.com/p2pderivatives/p2pderivatives-client/) as well

File: meeting/meeting_october_6_2020.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+  * Once this is done, I will be opening an ECDSA adaptor sig branch which will be permanent (until merged) and replace `new-temp-everything` branch
+* BIP 340 support for rust-secp256k1
+  * https://github.com/rust-bitcoin/rust-secp256k1/pull/237
+
+## Oracle Specifications
+
+* [URI format proposal](https://github.com/discreetlogcontracts/dlcspecs/pull/63)
+* [Descriptor proposal](https://github.com/discreetlogcontracts/dlcspecs/pull/55)
+* ["Oracle Address" proposal](https://github.com/discreetlogcontracts/dlcspecs/issues/99)
+
```
Comment (benthecarman) [inline meeting/meeting_october_6_2020.md:31]: I'd like to make a quick mention of #102

**Comment by Tibo-lg** [comment]
I would really love to get a final answer on whether enforcing change outputs in the fund transaction is a security issue or a convenience one (https://github.com/discreetlogcontracts/dlcspecs/pull/57#discussion_r484186590 and https://github.com/p2pderivatives/rust-dlc/pull/2#discussion_r494737495) cc @ariard

**Comment by ariard** [comment]
Sure @Tibo-lg let's talk about this during tmrw meeting 👍

**Pull request merged**